_signatures/Aaron_Turner.md

```diff
@@ -0,0 +1,6 @@
+---
+  name: Aaron Turner
+  link: https://www.twitter.com/synfinatic
```

Comment:
Hi Aaron, we need quotation marks around this URL.

http://stackoverflow.com/questions/11301650/how-to-escape-indicator-characters-i-e-or-in-yaml